QUICKSTART.md

```diff
@@ -231,7 +232,6 @@ public void handle(HttpExchange httpExchange) {
     serverSpan.setAttribute("http.target", "/resource");
     // Serve the request
     ...
-  } finally {
```

Comment:
why this? you need finally to ensure that serverSpan is always ended.

Reply:
the span is created inside the try. It is not be visible in the finally block.

Reply:
good point. the span declaration should happen outside the `try`, I think. And the finally should check for nullity.

Reply:
Whoops, sorry! I thought you removed the line accidentally.
Yep, the Quickstart should demonstrate proper usage as described by @jkwatson then.

Reply:
Since this was merged already I fixed it in #1064.